tools/startup/startup-basics-only.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+readonly DOWNLOAD_DIR_NAME="downloads_$(date +%Y%m%d_%H%M%S)"
+readonly DOWNLOAD_DIR_ABS_PATH="$(pwd)/$DOWNLOAD_DIR_NAME"
+readonly BASE_ABS_PATH=${HOME}/base
+
+sudo apt update
+echo "Checking for git installing"
+if [[ $(which git; echo $?) == "1" ]]; then
+    echo "Installing git"
+    sudo apt install git
+    echo "git is installed"
+fi
+
+echo "Checking for curl installing"
+if [[ $(which curl; echo $?) == "1" ]]; then
+    echo "Installing curl"
+    sudo apt install curl
+    echo "curl is installed"
+fi
+
+echo "Checking for internet connection"
```

Comment:
Perhaps this should be first? Assuming we have curl.

Reply:
This moved higher up.

Reply:
What happens if internet is unavailable?
Does the `sudo apt install curl` exit the script?
If internet is unavailable, does it make sense to run `sudo apt install curl`?

Reply:
Yes, you are right. It doesn't make sense to run `sudo apt install curl` if the internet is unavailable.
But we also can't check internet connection if `curl` isn't installed. 
I suggest in that case if `curl` isn't installed to print a message and exit the script:
```
Please ensure internet connection, install curl (sudo apt-get update && sudo apt-get install curl) and re-run the script.
```